tests/generator/ErrorTests.cs

```diff
@@ -149,6 +149,28 @@ public void BI1049 ()
 			bgen.AssertError (1049, "Could not unbox type String from NSNumber container used on member BindAs1049ErrorTests.MyFooClass.StringMethod decorated with [BindAs].");
 		}
 
+		[Test]
+		public void GH6863_property ()
+		{
+			var bgen = new BGenTool ();
+			bgen.Profile = Profile.iOS;
+			bgen.CreateTemporaryBinding (File.ReadAllText (Path.Combine (Configuration.SourceRoot, "tests", "generator", "ghissue6863_property.cs")));
+			bgen.AssertExecuteError ("build");
+			bgen.AssertError (1071, "The BindAs type for the member \"GH6863_property.MyFooClass.StringProp\" must be an array when the member's type is an array.");
+		}
+
+        
+		[Test]
+		public void GH6863_method ()
+		{
+			var bgen = new BGenTool ();
+			bgen.Profile = Profile.iOS;
+			bgen.CreateTemporaryBinding (File.ReadAllText (Path.Combine (Configuration.SourceRoot, "tests", "generator", "ghissue6863_method.cs")));
+			bgen.AssertExecuteError ("build");
+			bgen.AssertError (1072, "The BindAs type for the parameter \"id_test\" in the method \"GH6863_method.MyFooClass.StringMethod\" must be an array when the parameter's type is an array.");
+		}
+       
```

Comment:
Is it possible to have both the property and method tests in a single test function? Also there is probably something obvious here, but where does the "GH6863" come from?